R/annotate_intrachromosomal_sv.R

```diff
@@ -0,0 +1,124 @@
+#' @title Annotate intrachromosomal SV.
+#'
+#' @description Annotate the intrachromosomal events in a data frame of SV breakpoints.
+#'
+#' @details Specify a data frame with SVs (preferably the output from get_manta_sv() to the `sv_data` parameter and get back the same data frame with SV annotations.
+#'
+#' @param sv_data A data frame of SVs. This should be the output of get_manta_sv.
+#' The required columns are columns for both chromosomes, coordinates and strand plus SOMATIC_SCORE and tumour_sample_id.
```

Comment:
Perhaps, also add that `manta_name` is a required column?

Reply:
Added this, thanks!